sormas-ui/src/main/java/de/symeda/sormas/ui/caze/BulkCaseDataForm.java

```diff
@@ -206,8 +207,18 @@ protected void addFields() {
 				Arrays.asList(CaseBulkEditData.SURVEILLANCE_OFFICER),
 				Arrays.asList(true),
 				null);
-			List<UserReferenceDto> assignableCaseResponsibles =
-				FacadeProvider.getUserFacade().getUserRefsByDistrict(singleSelectedDistrict, UserRight.CASE_RESPONSIBLE);
+			List<Disease> selectedDiseases = this.selectedCases.stream().map(c -> c.getDisease()).collect(Collectors.toList());
+			List<UserReferenceDto> assignableCaseResponsibles = null;
+
+			if (selectedDiseases.size() == 1) {
```

Comment:
please review this logic, it does not seem to work when multiple cases having the same disease are selected